lib/inspec/json_profile_summary.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Inspec
+  #
+  # Inspec::JsonProfileSummary takes in certain information to identify a
+  # profile and then produces a JSON-formatted summary of that profile. It can
+  # return the results to STDOUT or a file. It is currently used in several
+  # places in the CLI such as `json`, `archive` and `artifact`.
+  #
+  #
+  module JsonProfileSummary
```

Comment:
This is a pretty specific thing to put directly into `Inspec::`. It's not wrong by any means, but much of the code has modules arranged like `Inspec::Category::Thing` 
There is a `Util` module that this could go inside.